src/mouse_handler_base.cpp

```diff
@@ -349,7 +352,8 @@ void mouse_handler_base::left_drag_end(int /*x*/, int /*y*/, const bool browse)
 
 void mouse_handler_base::mouse_wheel(int scrollx, int scrolly, bool browse)
 {
-	auto [x, y] = sdl::get_mouse_location();
+	float x, y;
```

Comment:
What's happening here? What was wrong with `get_mouse_location`?

Reply:
It's been a while, so not sure. Probably left over as part of dealing with it being floats now.